source/beamline/SXRMB/SXRMBBrukerDetector.cpp

```diff
@@ -28,6 +28,7 @@ SXRMBBrukerDetector::SXRMBBrukerDetector(const QString &name, const QString &des
 
 	// Stuff specialized by SXRMBBrukerDetector
 	deadTimeControl_ = new AMReadOnlyPVControl("Dead Time", "mca1606-B10-03:mca1.DTIM", this);
+	allControls_->addControl(deadTimeControl_);
 	connect(deadTimeControl_, SIGNAL(connected(bool)), this, SLOT(onDeadTimeControlConnected(bool)));
```

Comment:
By adding `deadTimeControl_` to the `allControls_` set, you no longer need the specific connect from the dead time control.  Probably a change of:

```c++
connect(this, SIGNAL(connected(bool)), this, SLOT(onDeadTimeControlConnected(bool)));
```